routes/middleware/get_places.js

```diff
@@ -1,36 +1,27 @@
-const request = require('request');
+const axios = require('axios');
 
-module.exports = (req, res, next) => {
-  const url = 'https://nazareth-open-tourism-platform.herokuapp.com/api/v1/places';
+module.exports = async (req, res, next) => {
+  const url = `${process.env.URI}/places`;
+  const placesResponse = await axios.get(url);
```

Comment:
and what do i do for error handling should i redirect the user to somewhere else orr??? 

cause for now i'm adding a next() so that would show an empty dropdown list :grimacing: any idea?

Reply:
Hmmm.. There should be some kind of error page I guess :man_shrugging: The only time I can imagine it happening is if the OTP is down, in which case nothing else on NE will work.